content/docs/cve/cveIn-PersonExamProcess.md

```diff
@@ -109,11 +110,11 @@ Will not receive anything unless the team would like to either print or email th
 
 ## VEC Upload, Licenses and Upgrades:
 
-Once the VEC has uploaded the batch file from the session, the FCC will process to review the applications and for most approve and issue callsigns the same day. Once this happens, the applicant should get an email from Exam Tools and the FCC on Monday for weekend exams, if everything goes well with EBF upload. The email from the FCC comes overnight with a link to the official license. The applicant should download the license from the link, maybe rename it something other than what it is called (just a long string of numbers starting with their FRN) and save it in a place where they can find it again. They may choose to print out a copy and keep it with their radio.
+Once the VEC has uploaded the batch file from the session, the FCC will process to review the applications and for most approve and issue call signs the same day. Once this happens, the applicant should get an email from ExamTools and the FCC on Monday for weekend exams, if everything goes well with EBF upload. The email from the FCC comes overnight with a link to the official license. The applicant should download the license from the link, maybe rename it something other than what it is called (just a long string of numbers starting with their FRN) and save it in a place where they can find it again. They may choose to print out a copy and keep it with their radio.
 
 Those who upgraded can use their existing call sign appended with /AG (General) or /AE (Extra) right now to try out their new privileges. Those who do not have a license will need to wait until their call sign appears in the ULS which should be sometime during the day. Occasionally, applicants need to wait an additional day or two while their application is in review.
 
-The FCC issues callsigns during business hours Monday through Friday. If an applicant tests early enough on a weekday, the applicant could possibly see their application processed the same day, but should expect 5-7 days processing time.
+The FCC issues call signs during business hours Monday through Friday. If an applicant tests early enough on a weekday, the applicant could possibly see their application processed the same day, but should expect 5-7 days processing time.
```

Comment:
This isn't actually correct; ULS itself runs from something like 3:30 am to 11pm Eastern time, M-F.  Each of the VECs have different processing times and schedules, though.  I'd also remove the "same day call sign" reference.